### PR TITLE
separate custom styles from bootstrap variable overwrites

### PR DESCRIPTION
**Pull request by casaper**
- each section has its scss file in the layout directory
- everything is imported in application.scss

## Story
Link to trello story

## Goals
- [x] Story is tested
- ~Acceptance criterias are met~ none listed
- [ ] Code is reviewed
- [x] Travis is green
- [x] Story is on resolved in trello
- [x] Readme is updated if needed
- [x] Styled with Bootstrap v3
- [x] Mobile view is usable
- ~Seeds are available~
- ~German translations available~

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,4 +1,9 @@
 @import 'bootstrap_overwrites';
+
 @import 'bootstrap-sprockets';
 @import 'bootstrap';
-@import 'main';
+@import 'bootstrap/variables';
+
+@import 'layout/navbar.scss';
```
Comment (kitsane) [inline app/assets/stylesheets/application.scss]: `.scss`not needed

File: app/assets/stylesheets/layout/_navbar.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    border-color: $black;
+  }
+  .icon-bar {
+    background-color: $black;
+  }
+  .navbar-brand {
+    height: $navbar-height - $padding-xs-vertical * 2;
+    padding: $padding-xs-vertical $padding-xs-horizontal;
+
+    .brand-image {
```
Comment (kitsane) [inline app/assets/stylesheets/layout/_navbar.scss]: The navbar image is to large now, compared to develop branch

File: app/assets/stylesheets/layout/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.navbar-top {
```
Comment (kitsane) [inline app/assets/stylesheets/layout/_navbar.scss]: Did your rewrite and/or add some styles? What is the reason behind?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (casaper) [inline app/assets/stylesheets/application.scss]: forgotten corpse ... removed

File: app/assets/stylesheets/layout/_navbar.scss (hunk 2 above)
Comment (casaper) [inline app/assets/stylesheets/layout/_navbar.scss]: i was adapting it to look like some old version of develop, so my bad. In this case the css also doesn't need to be so bl... complicated

File: app/assets/stylesheets/layout/_navbar.scss (hunk 3 above)
Comment (casaper) [inline app/assets/stylesheets/layout/_navbar.scss]: namespaceing for the sake of it. but I guess your right. removed

**Pull request merged**